db/migrate/20150911195321_add_budget_roles.rb

```diff
@@ -0,0 +1,24 @@
+class AddBudgetRoles < ActiveRecord::Migration
+  def change
+    User.with_email_role_slug!(
+      ENV['NCR_BA61_TIER1_BUDGET_MAILBOX'] || 'communicart.budget.approver@gmail.com',
+      'BA61_tier1_budget_approver',
+      'ncr'
+    )
+    User.with_email_role_slug!(
+      ENV['NCR_BA61_TIER2_BUDGET_MAILBOX'] || 'communicart.ofm.approver@gmail.com',
+      'BA61_tier2_budget_approver',
+      'ncr'
+    )
+    User.with_email_role_slug!(
+      ENV['NCR_BA80_BUDGET_MAILBOX'] || 'communicart.budget.approver@gmail.com',
+      'BA80_budget_approver',
+      'ncr'
+    )
+    User.with_email_role_slug!(
+      ENV['NCR_OOL_BA80_BUDGET_MAILBOX'] || 'communicart.budget.approver@gmail.com',
+      'OOL_BA80_budget_approver',
+      'ncr'
+    )
+  end
+end
```

Comment:
> provide base fixtures for a clean checkout

That's done as part of the [bootstrap](https://github.com/18F/C2/blob/36842b72024c5b387a7f96c8dbbabbcfa6f6ed5d/script/bootstrap#L13).

> If `db:seed` were part of a regular deploy

We could certainly add this, but would be nice if we didn't need those environment variables set indefinitely.

> we never seed an existing, working db

We don't, but that's because we don't currently have any seed data. If we do, ideally running `rake db:seed` should be idempotent (by using `find_or_create_by`).